packages/shared/src/components/filters/TagCategoryDropdown.tsx

```diff
@@ -1,35 +1,52 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { TagCategory } from '../../graphql/feedSettings';
 import ArrowIcon from '../icons/Arrow';
 import CategoryButton from './CategoryButton';
 import TagButton from './TagButton';
 import {
+  BaseTagCategoryDetails,
   TagCategoryDetails,
   TagCategoryDetailsContent,
+  BaseTagCategorySummary,
   TagCategorySummary,
 } from './common';
 import { TagActionArguments } from '../../hooks/useTagAndSource';
 
-type TagCategoryDropdownProps = {
+interface TagCategoryDropdownProps {
+  version?: string;
   tagCategory: TagCategory;
   followedTags?: Array<string>;
   blockedTags?: Array<string>;
   onFollowTags?: ({ tags, category }: TagActionArguments) => void;
   onUnfollowTags?: ({ tags, category }: TagActionArguments) => void;
   onUnblockTags?: ({ tags }: TagActionArguments) => void;
-};
+}
 
 export default function TagCategoryDropdown({
+  version,
   tagCategory,
   followedTags,
   blockedTags,
   onFollowTags,
   onUnfollowTags,
   onUnblockTags,
 }: TagCategoryDropdownProps): ReactElement {
+  const isV2 = version === 'v2';
```

Comment:
What exactly are the versions for this modal? 
Or is this related to when it renders inside the feed filter onboarding?

Reply:
This is a component used by the feed filter onboarding version 1, but then we can easily reuse it for the new feed filter settings when logged-in which is why I applied it here. Now that you mentioned it, I feel like it is more appropriate if we will name the variable `isOnboarding` as this is not something related to AB testing. WDYT?

Ref: onboarding version 1
![image](https://user-images.githubusercontent.com/13744167/194893326-d2014cc6-8f35-4529-8165-8b199d10651f.png)

Ref: feed filters menu (always like this, no AB testing)
![image](https://user-images.githubusercontent.com/13744167/194894016-2a13f9c3-db89-4dcf-9c9d-8db0e2873f85.png)

Reply:
On another note, we need to remove the icons in this version

Reply:
I think Ido has delete everything out in the DB as I don't see it in prod anymore.